### PR TITLE
add or change comment for classes

### PR DESCRIPTION
**Pull request by ramsayleung**
1. capitalize word in the beginning
2. add comment for classes which has no comment
3. add demonstration for test class
4. unify comment in a same class

### CODE REVIEW

File: src/main/java/io/reactivesw/category/application/controller/CategoryController.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
 public class CategoryController {
 
   /**
-   * log.
+   * logger.
    */
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/application/controller/CategoryController.java]: comment should start with uppercase letter.

File: src/main/java/io/reactivesw/category/application/controller/HealthController.java [hunk 2]
```diff
@@ -15,18 +15,18 @@
 public class HealthController {
 
   /**
-   * log.
+   * logger.
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/application/controller/HealthController.java]: comment should start with uppercase letter.

File: src/main/java/io/reactivesw/category/application/model/mapper/CategoryMapper.java [hunk 3]
```diff
@@ -18,13 +18,13 @@
...
   /**
-   * constructor.
+   * Constructor.
    */
   private CategoryMapper() {
   }
 
   /**
-   * Map entity to category category.
+   * Convert entity to category category.
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/application/model/mapper/CategoryMapper.java]: two category.

File: src/main/java/io/reactivesw/category/application/model/mapper/LocalizedStringMapper.java [hunk 4]
```diff
@@ -70,13 +70,14 @@ public static LocalizedString toModelDefaultNull(Set<LocalizedStringValue>
 
 
   /**
-   * Model to entity default null set.
+   * Model to entity
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/application/model/mapper/LocalizedStringMapper.java]: should end with a dot.

File: src/main/java/io/reactivesw/category/application/service/CategoryApplication.java [hunk 5]
```diff
@@ -16,8 +16,9 @@
  */
 @Service
 public class CategoryApplication {
+
   /**
-   * log.
+   * logger.
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/application/service/CategoryApplication.java]: comment should start with uppercase letter.

File: src/main/java/io/reactivesw/category/application/service/EventPublisher.java [hunk 6]
```diff
@@ -25,13 +25,13 @@
 public class EventPublisher {
 
   /**
-   * log.
+   * logger.
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/application/service/EventPublisher.java]: comment should start with uppercase letter.

File: src/main/java/io/reactivesw/category/domain/service/EventMessageService.java [hunk 7]
```diff
@@ -24,7 +24,7 @@
 @Service
 public class EventMessageService {
   /**
-   * log.
+   * logger.
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/domain/service/EventMessageService.java]: comment should start with uppercase letter.

File: src/main/java/io/reactivesw/category/domain/service/update/SetDescriptionService.java [hunk 8]
```diff
@@ -15,7 +15,7 @@
 @Service(value = CategoryActionUtils.SET_DESCRIPTION)
 public class SetDescriptionService implements Updater<Category, UpdateAction> {
   /**
-   * set description.
+   * Set description.
    * @param entity E
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/domain/service/update/SetDescriptionService.java]: need a blank line here.

File: src/main/java/io/reactivesw/category/domain/service/update/SetMetaDescriptionService.java [hunk 9]
```diff
@@ -14,7 +14,7 @@
 @Service(value = CategoryActionUtils.SET_META_DESCRIPTION)
 public class SetMetaDescriptionService implements Updater<Category, UpdateAction> {
   /**
-   * set meta description.
+   * Set meta description.
    * @param entity E
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/domain/service/update/SetMetaDescriptionService.java]: need a blank line here.

File: src/main/java/io/reactivesw/category/infrastructure/exception/CategoryExceptionHandler.java [hunk 10]
```diff
@@ -16,7 +16,7 @@
 public class CategoryExceptionHandler extends ExceptionHandler implements HandlerExceptionResolver {
 
   /**
-   * exception handle method.
+   * Exception handle method.
    * @param request HttpServletRequest
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/infrastructure/exception/CategoryExceptionHandler.java]: need a blank line here.

File: src/main/java/io/reactivesw/category/infrastructure/util/ReferenceTypes.java [hunk 11]
```diff
@@ -32,20 +32,20 @@
...
   /**
-   * enum value.
+   * Enum value.
    */
   private String value;
 
   /**
-   * private constructor.
+   * Private constructor.
    * @param value enum value
```
Comment (davis999) [inline src/main/java/io/reactivesw/category/infrastructure/util/ReferenceTypes.java]: need a blank line here.

File: src/main/java/io/reactivesw/category/application/controller/CategoryController.java (hunk 1 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/application/controller/CategoryController.java]: capitalized  logger.

File: src/main/java/io/reactivesw/category/application/model/mapper/CategoryMapper.java (hunk 3 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/application/model/mapper/CategoryMapper.java]: deleted a category.

File: src/main/java/io/reactivesw/category/application/service/CategoryApplication.java (hunk 5 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/application/service/CategoryApplication.java]: capitalize logger.

File: src/main/java/io/reactivesw/category/application/service/EventPublisher.java (hunk 6 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/application/service/EventPublisher.java]: capitalize logger.

File: src/main/java/io/reactivesw/category/domain/service/EventMessageService.java (hunk 7 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/domain/service/EventMessageService.java]: capitalize logger.

File: src/main/java/io/reactivesw/category/domain/service/update/SetDescriptionService.java (hunk 8 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/domain/service/update/SetDescriptionService.java]: added a blank line.

File: src/main/java/io/reactivesw/category/domain/service/update/SetMetaDescriptionService.java (hunk 9 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/domain/service/update/SetMetaDescriptionService.java]: added a blank line.

File: src/main/java/io/reactivesw/category/infrastructure/exception/CategoryExceptionHandler.java (hunk 10 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/infrastructure/exception/CategoryExceptionHandler.java]: added blank line.

File: src/main/java/io/reactivesw/category/infrastructure/util/ReferenceTypes.java (hunk 11 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/infrastructure/util/ReferenceTypes.java]: added a blank line.

File: src/main/java/io/reactivesw/category/application/model/mapper/LocalizedStringMapper.java (hunk 4 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/application/model/mapper/LocalizedStringMapper.java]: added a dot.

File: src/main/java/io/reactivesw/category/application/controller/HealthController.java (hunk 2 above)
Comment (ramsayleung) [reply src/main/java/io/reactivesw/category/application/controller/HealthController.java]: capitalized logger.

**Review by davis999 — APPROVED** [review]

**Pull request merged**